src/components/Table/GridTable.tsx

```diff
@@ -195,21 +196,25 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
     api.init(persistCollapse, virtuosoRef, rows);
     api.setActiveRowId(activeRowId);
     api.setActiveCellId(activeCellId);
+    api.tableState.initSortState(props.sorting, columns);
     return api;
   }, [props.api]);
 
   const style = resolveStyles(maybeStyle);
+  const { tableState } = api;
+  const { sortConfig, setSortKey } = tableState;
+  const { on: sortOn } = sortConfig ?? ({} as GridSortConfig);
+  const caseSensitive = sortConfig?.on === "client" ? !!sortConfig.caseSensitive : false;
```

Comment:
Thought: In general the destructure of `tablestate` and these `sortOn` / `caseSensitive` values being done directly in the render method worry me just a little bit b/c, b/c they're not wrapped in an observer, we will get no reactivity if they change.

Granted, it seems unlikely a page would actually change any of these, but, dunno, just fwiw would be nice to see these accessed in a `useComputed`. Not required if it's not affecting anything currently.

Reply:
Yea, part of me did this on purpose as I don't think we care to make them reactive properties. But who knows in the future if we'll give the user an option to edit the sorting behavior later.. 🤷 So I'm good with updating.